.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: java
+sudo: false
+script: mvn -Pdev verify
+jdk:
+  - oraclejdk8
+  - openjdk8
+env:
+  global:
+    - JAVA_OPTS="-Xmx1024M -Xms512M"
+notifications:
+  irc:
+    channels:
+      - "irc.freenode.org#jena"
```

Comment:
Having the IRC channel notified.

And if anyone clones this and adds it to jarvis, we get it twice.